packages/connectivity/src/scp-cf/destination/destination-accessor-jwks.spec.ts

```diff
@@ -1,34 +1,24 @@
 import nock from 'nock';
 import {
+  customSubscriberUserToken,
   destinationName,
   destinationSingleResponse,
-  jku,
   mockInstanceDestinationsCall,
   mockServiceBindings,
   mockServiceToken,
   mockSingleDestinationCall,
   mockSubaccountDestinationsCall,
-  mockVerifyJwt,
   oauthMultipleResponse,
-  providerServiceTokenPayload,
-  signedJwtForVerification,
-  subscriberServiceTokenPayload,
-  subscriberUserPayload
+  providerServiceToken
 } from '../../../../../test-resources/test/test-util';
-import * as jwt from '../jwt';
-import { responseWithPublicKey } from '../jwt.spec';
 import { DestinationFetchOptions } from './destination-accessor-types';
-import {
-  alwaysProvider,
-  alwaysSubscriber
-} from './destination-selection-strategies';
+import { alwaysProvider } from './destination-selection-strategies';
 import { getDestination } from './destination-accessor';
 import { DestinationConfiguration } from './destination';
 
-describe('custom jwt via jwks property on destination', () => {
+describe('custom JWTs', () => {
```

Comment:
I'm missing the context to understand why and how this test changes in this PR.

Reply:
The `isXsuaaToken()` function checks for `enhancer` instead of the `jku` header property, so the tests are not relevant anymore.